### PR TITLE
Update docs for running frontend apps to include govuk-docker

### PR DESCRIPTION
**Pull request by vanitabarrett**
[View rendered file here](https://github.com/alphagov/govuk-developer-docs/blob/9894ebcabe6178831ff04722e3c565814f4494e6/source/manual/local-frontend-development.html.md)

### CODE REVIEW

File: source/manual/local-frontend-development.html.md [hunk 1]
```diff
@@ -9,67 +9,80 @@ last_reviewed_on: 2019-07-08
...
+
 ./startup.sh --live
 ```
 
-## Running a frontend app against changes in `static`
+## Using govuk-docker
+This assumes that you have already installed and setup [govuk-docker].
 
-Make changes to the Gemfile of the frontend app you're working on as per [changes to govuk_publishing_components above](#running-a-frontend-app-against-changes-in-govuk_publishing_components), then set in `development.rb`:
+If you are simply making changes to a frontend app and nothing else, you can view these changes by running the following:
```
Comment (issyl0) [inline source/manual/local-frontend-development.html.md]:
```suggestion
If you are making changes to a frontend app and nothing else, you can view these changes by running the following:
```

File: source/manual/local-frontend-development.html.md [hunk 2]
```diff
@@ -9,67 +9,80 @@ last_reviewed_on: 2019-07-08
 review_in: 6 months
 ---
 
-Make the following changes to your setup if you are making changes to frontend apps and need to view those changes locally.
-
-In the instructions below we are running the [frontend] app, but this would work for any of the other frontend apps such as [government-frontend].
-
-## Running just a frontend app against live data
+## Using startup scripts
+If you are simply making changes to a frontend app and nothing else, you can view these changes by running the application `./startup.sh` script:
```
Comment (issyl0) [inline source/manual/local-frontend-development.html.md]:
```suggestion
If you are making changes to a frontend app and nothing else, you can view these changes by running the application `./startup.sh` script:
```

**Comment by vanitabarrett** [comment]
Thanks both! Kelvin has actually added some govuk-docker steps already, so I'm going to close this PR and open a new one based off his changes.

**Comment by vanitabarrett** [comment]
Superseded by https://github.com/alphagov/govuk-developer-docs/pull/2210

**Pull request closed without merging**